src/I18nExtension.php

```diff
@@ -18,29 +19,48 @@
 
 class I18nExtension extends AbstractExtension
 {
-    /**
-     * {@inheritdoc}
-     */
-    public function getTokenParsers()
-    {
-        return [new TransTokenParser()];
-    }
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getTokenParsers()
+	{
+		return [new TransTokenParser()];
+	}
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getFilters()
-    {
-        return [
-            new TwigFilter('trans', 'gettext'),
-        ];
-    }
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getFilters()
+	{
+		return [
+			new TwigFilter('trans', '\PhpMyAdmin\Twig\Extensions\I18nExtension::translate'), /* Note, the filter does not handle plurals */
+		];
+	}
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getName()
-    {
-        return 'i18n';
-    }
+	/**
+	 * {@inheritdoc}
+	 */
+	public function getName()
+	{
+		return 'i18n';
+	}
+
+	/**
+	 * Translate a GetText string via filter
+	 *
+	 * @param string| $message The message to translate
+	 * @param string|null $domain The GetText domain
+	 *
+	 * @return string
+	 */
+	public static function translate($message, $domain = null)
+	{
+		/* If we don't have a domain, assume we're just using the default */
+		if ($domain === null) {
+			return gettext($message);
+		}
+		else { /* Otherwise specify where the message comes from */
```

Comment:
Please run `composer run phpcbf` and composer run phpcs`

Also it would be awesome if you could add an unit test

Reply:
I've tried using the phpcbf and for a mixture of reasons it's just not working for me. It varies between not being able to add sniffs and not finding the correct path. I've had a play around and I'm not really sure what's wrong with it. I'm not really familiar with it, so that may be why I'm not sure.

Likewise, unit tests are not really something I've ever done so I'm afraid I'm not really sure what to do there. Sorry.

Reply:
Okay, I can work on unit tests then ;)